course_catalog/utils.py

```diff
@@ -0,0 +1,18 @@
+""" Utils for course catalog """
+from profiles.utils import generate_filepath
+
+
+def learning_path_image_upload_uri(instance, filename):
+    """
+    upload_to handler for user-created Learning Path image
+    """
+    return generate_filepath(
```

Comment:
Since this PR pulls this into a different module, can you move `generate_filepath` and its tests into `open_discussions/utils.py` so it's obvious this is a more generic method?

Reply:
Moved into `open_discussions/utils.py`